endpoints/put_cookies.md

```diff
@@ -26,7 +26,6 @@ Updates cookies. The request body has the following shape:
 ```
 
 - `account_id` is the unique identifier of the account
-- `id` is the unique identifier of the cookie
```

Comment:
Why are you removing this line? This is an update. It requires a cookie id to update.

Reply:
It's a duplicate of line 37 above "initiator". 

<img width="1029" alt="Screen Shot 2022-01-06 at 4 26 22 PM" src="https://user-images.githubusercontent.com/75861275/148454458-6f60ac94-ce51-4936-a132-ce4213e45d35.png">

Reply:
I left the one on line 37 because that's where it's placed in the all other request bodies on this page.

Reply:
Thanks!